cli/tests/test_upgrade_smoke_static.py

```
@@ -1030,9 +1030,152 @@ def test_posix_resolver_bootstraps_recovery_under_fixed_mutator_lease() -> None:
+    tmp_path: Path,
+) -> None:
+    resolver = (ROOT / "scripts/upgrade.sh").read_text(encoding="utf-8")
+    version_match = re.search(
+        r'readonly UV_BOOTSTRAP_VERSION="([^"]+)"',
+        resolver,
+    )
+    maximum_match = re.search(
+        r'readonly UV_BOOTSTRAP_MAX_BYTES="([^"]+)"',
+        resolver,
+    )
+    assert version_match is not None and maximum_match is not None
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)